main.go

```diff
@@ -47,6 +49,34 @@ func main() {
 			Usage:  "Password for authing against jenkins",
 			EnvVar: "JENKINS_PASSWORD",
 		},
+		cli.StringFlag{
+			Name:   "cloudops-pulse-prefix",
+			Usage:  "Pulse route to listen to.",
+			Value:  "cloudops.v1.deploy",
+			EnvVar: "CLOUDOPS_PULSE_PREFIX",
+		},
+		cli.StringFlag{
+			Name:   "pulse-queue",
```

Comment:
Looks like this setting isn't used anywhere.

Reply:
It should be passed to the pulse connection. I wrote this a while ago, but I suspect I wanted to use an anonymous queue for testing locally.